### PR TITLE
style: check file eol

### PR DESCRIPTION
**Pull request by trim21**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -5,6 +5,13 @@ repos:
   - id: end-of-file-fixer
   - id: trailing-whitespace
   - id: check-added-large-files
+  - id: mixed-line-ending
+    args: [--fix=lf]
+    exclude_types: [batch]
+  - id: mixed-line-ending
+    args: [--fix=crlf]
+    types: [batch]
+
```
Comment (Delgan) [inline .pre-commit-config.yaml:14]: I don't think this repository will ever contain batch files, can we just force LF for every files please?

```suggestion
  - id: mixed-line-ending
    args: [--fix=lf]**
```

Comment (trim21) [reply .pre-commit-config.yaml:14]: it does https://github.com/Delgan/loguru/blob/master/docs/make.bat

Comment (Delgan) [reply .pre-commit-config.yaml:14]: Haha, my bad.

**Comment by Delgan** [comment]
Thanks!

File: .pre-commit-config.yaml (hunk 1 above)
Comment (Delgan) [reply .pre-commit-config.yaml:14]: Yes, I agree. I should have run `sphinx-build` with the `--no-batchfile` argument.

**Pull request merged**